src/impy/models/epos.py

```diff
@@ -45,29 +46,26 @@ class EposLHC(MCRun):
         + "/releases/download/zipped_data_v1.0/epos_v001.zip"
     )
 
-    def __init__(self, evt_kin, *, seed=None):
-        import impy
-
-        super().__init__(seed)
+    def _once(self, ecm_max=1000 * TeV):
+        from impy import debug_level
```

Comment:
Does this imply that the debug level cannot change in runtime? It is important if you search for something that goes wrong at event 12345 but don't want to see the output of 12344 events.

Reply:
If you pass the debug level to fortran in a function that is called only at init, then you cannot change the debug level at run time. This is how all of our models are currently implemented. We never had an interface to change the debug level after init.

Reply:
Regarding this:
```
impy.debug_level = 0
m1 = Epos() # do something

impy.debug_level = 5
m2 = Phojet() # do something different
```
I thought this works, this is why `debug_level` is imported freshly from top level `impy` (and to avoid some circular import issue). Did you test it and found it does not work?